sdk/iot/core/README.md

```diff
@@ -55,6 +55,43 @@ int main()
 }
 ```
 
+### Properties
+
+Included in the IoT SDK are helper functions to form and manage properties for IoT Hub services. Functionality starts by using the `az_iot_hub_client_properties_init()` API. The user is free to intitialize using an empty but appropriately sized span to later append properties or an already populated span containing a properly formated property buffer. "Properly formatted" properties follow the form `{key}={value}&{key}={value}`.

```

Comment:
Implementation starts by using the

Reply:
The user is free to intitialize using an empty, but appropriately sized span, to later